packages/driver/src/cy/commands/request.ts

```diff
@@ -222,7 +227,7 @@ export default (Commands, Cypress, cy, state, config) => {
 
       // only set json to true if form isnt true
       // and we have a valid object for body
-      if ((options.form !== true) && isValidJsonObj(options.body)) {
+      if ((options.form !== true) && isValidJsonObj(options.body, bodyIsExplicitlyDefined)) {
```

Comment:
```suggestion
      if ((options.form !== true) && isValidBody(options.body, bodyIsExplicitlyDefined)) {
```

Reply:
is `body` implicitly set to `null` when it isn't defined and thats why we need the `bodyIsExplicitlyDefined` flag?

Reply:
> is `body` implicitly set to `null` when it isn't defined and thats why we need the `bodyIsExplicitlyDefined` flag?

Yes, `options.body` is always initialized as `null` in `REQUEST_DEFAULTS` (_request.ts:21_)